### DISCUSSION TITLE
A review of Koalas. Comments welcome.

### DISCUSSION THREAD

**Discussion by ChuckConnell** [databricks/koalas#2192] [Show and tell]
https://medium.com/@chuck.connell.3/pandas-on-databricks-via-koalas-a-review-9876b0a92541

Thanks to @itholic for his help.

**Comment by itholic** [comment]
Let me check soon and will leave some comments if there is any opinion !!

Thanks for your dedication for writing this blog post :-)

**Comment by itholic** [comment]
Hi, @ChuckConnell !!

We noticed some improvements in Koalas from your blog post, so I opened two tickets below.

(They are opened in Apache Spark side, since Koalas is ported into PySpark under the name "pandas APIs on Spark".)
- add `errors` argument for `to_numeric`. (https://issues.apache.org/jira/browse/SPARK-36609)
- add `thousands` argument for `read_csv`. (https://issues.apache.org/jira/browse/SPARK-36610)

Koalas can be improved one step further, thanks to your contribution :-)

Thank you!!
